### DISCUSSION TITLE
GoPro Hero 11 Davinci Color Management

### DISCUSSION THREAD

**Discussion by zacfpv** [gopro/labs#880] [Q&A] [answered]
I'm currently running the labs Firmware on GoPro Hero 11 and been using the WIDE gamut and LOGB = 400 curve. I currently plan to do the color grade in resolve and what would be color management settings for the WIDE gamut and Log curve? Should I use the color space transform or just the LUT from GoPro? Could you point me in the right direction?

**Comment by Deadcode85** [comment, marked as answer]
With Wide, LOGB=200,-128 you can use CST with REC2020 - Fuji Flog as in input color space and gamma with pretty decent result.
I created a conversion LUT for Wide, LOGB=400, which transforms the footage into SLOG2-SGamut, you can download it from here:
[GP400_Wide_to_Slog2Sgamut.cube.zip](https://github.com/gopro/labs/files/15170839/GP400_Wide_to_Slog2Sgamut.cube.zip)

The clipping point is not the same as the SLOG2, but most of the LUTS, input device transforms, ACES conversions will look very close to what a sony camera looks like. (Conversion made with A7 III, matched WB, tested through in the past 6+ month)
If you want to use sony LUTs after my lut, keep in mind, GoPro LOGB400 with -1EV compensation will look the same as 
Sony A7III with +1,66EV if the metering mode is matrix and auto.

![slog-gopro_3 1 1](https://github.com/gopro/labs/assets/97214191/857924d4-5f0a-420f-b5c4-209041e9a7b2)

This is not an exact match, saturation, hue levels are matched, i didnt touch the luma of the colors, because it breaks the footage easily for marginally better results.

**Reply by zacfpv** [reply]
This really did help ;) Thanks

**Reply by lyricsdude** [reply]
very nice conversion lut :) I'm wondering if you also have a conversion lut only for the colors. from gopro flat to gopro flat s-gamut colors. That would be great :)

**Reply by Deadcode85** [reply]
> very nice conversion lut :) I'm wondering if you also have a conversion lut only for the colors. from gopro flat to gopro flat s-gamut colors. That would be great :)

Im not sure why you might need that, [but i created](https://drive.google.com/drive/folders/12CVSAhLxSVyODqgfEg0E_1f9djvLUBqq?usp=sharing) GoPro Flat to Slog2-Sgamut and GoPro Flat to GoProFlat-sgamut LUT's for you. Keep in mind i did this with LUTCALC and not with color matching colorcheckers this time. WB and Exposure adjustments may be required before applying the LUTs.

**Reply by lyricsdude** [reply]
it's not really working. but thank you for the effort. I edited the LUT with 3Dlutcreator and it works fine now :)

**Comment by RikshaDriver** [comment]
For LOGB=400 and Rec.709 color gamut or Native color Gamut, you can use ACES IDTs or Davinci YRGB transforms:

https://github.com/xtremestuff/protune-transforms

Just be aware the "F-Log" look is a crude hack as commented a few times in these discussions and elsewhere. Best to avoid it if you're going down the color management path.
